apps/volontulo/views/organizations.py

```diff
@@ -65,6 +67,19 @@ def post(request):
                 {'organization': Organization()}
             )
 
+        try:
+            validate_email(request.POST.get('address'))
+        except ValidationError:
+            messages.error(
+                request,
+                u"Podaj poprawny adres email."
```

Comment:
@iwogar it's mistake to use `'u` prefix in `python3` code. It's used here a lot, but we want to not make this mistake anymore.